README.md

```diff
@@ -202,6 +203,44 @@ Where "dir" is a subdirectory within the repo to push all the check files. Setup
 4. Create `updates/.gitkeep`.
 5. Enable GitHub Pages in your settings for your desired branch.
 
+#### SQL Storage (sqlite3/PostgreSQL)
+
+**[godoc: SQL](https://godoc.org/github.com/sourcegraph/checkup#SQL)**
+
+Postgres or sqlite3 databases can be used as storage backends.
+
+sqlite database file configuration:
+```json
+{
+	"provider": "sql",
+	"sqlite_db_file": "/path/to/your/sqlite.db"
+}
+```
+
+sqlite database file configuration:
+```json
+{
+	"provider": "sql",
+	"postgresql": {
+		"user": "postgres",
+		"dbname": "dbname",
+		"host": "localhost",
+		"port": 5432,
+		"password": "password",
+		"sslmode": "disable"
+	}
+}
+```
+
+The SQL engine used depends on which one is configured; if both sqlite and Postgres configurations are present, sqlite is used.
+
+For all database backends, the database must exist and a "checks" table should be created:
+
+```
+CREATE TABLE checks (name TEXT NOT NULL PRIMARY KEY, timestamp INT8, results TEXT);
```

Comment:
I usually (almost always, in fact) have an autoincrement ID field primary key for internal use, in case, say, a name is wrong and gets changed. Should we add such a field?